_sass/tools/_mixins.scss

```diff
@@ -45,3 +45,10 @@
   flex-wrap: wrap;
   align-items: flex-start;
 }
+
+@mixin display-flex-news {
```

Comment:
This is great! I'm wondering if you'll end up using this elsewhere because it seems sort of like a utility... Maybe it'd be a good idea to name it something more general so that you could use it somewhere other than the "News" section and it would still make sense!

Reply:
true true. I'll name it something more reusable